docs/guides/moodleapp/development/testing/unit-testing.md

```diff
@@ -17,7 +17,7 @@ You can also watch changes in your codebase to rerun tests using the `npm run te
 
 If you are using VSCode, you can use [the built-in debugger](https://code.visualstudio.com/Docs/editor/debugging) to run your tests and stop at breakpoints. The project comes with two tasks preconfigured:
 
-- “Jest All” will run your entire test suite. It’s the equivalent of running `npm test` from the command line.
+- “Jest All” will run your entire test suite. It's the equivalent of running `npm test` from the command line.
 - “Jest Current File” will run the test of the file you have opened in the editor. Like the `--watch` filter, this will be a partial match based on the file name.
```

Comment:
```suggestion
- `Jest Current File` will run the test of the file you have opened in the editor. Like the `--watch` filter, this will be a partial match based on the file name.
```